### ISSUE TITLE
autoscroll not working

### ISSUE THREAD

**Issue by Ingegneus** [jeffeb3/OctoPrint-TerminalMessaging#12]
when using the terminal messaging plugin, the autoscroll in the terminal doesn't work. the terminal is just empty like this 
![image](https://github.com/jeffeb3/OctoPrint-TerminalMessaging/assets/78076361/f673e1d6-8da6-427f-9f05-b2226302289d)
from time to time it glitches back into existence.

**Comment by jeffeb3** [comment]
Have you confirmed it works without the plugin? This plugin shouldn't affect what gets drawn. There are a limited number of messages it keeps, so if they are all filtered out, it will display nothing. That isn't a limit with this plugin.

**Comment by Ingegneus** [comment]
![image](https://github.com/jeffeb3/OctoPrint-TerminalMessaging/assets/78076361/2f00f41f-d602-478f-8e83-8c4ab2b0dcae)
this is how it looks when the plugin is deactivated, and the output doesn't glitch in and out as it does when the plugin is enabled

> There are a limited number of messages it keeps, so if they are all filtered out, it will display nothing

what do you mean by that?

**Comment by jeffeb3** [comment]
I'm going off of memory (I don't have octoprint installed ATM). But there are something like 500 messages to/from the mcu recorded in a log. When you hit "surpress temperature messages", it filters out those temp responses. If all 500 of the stored messages are temp responses, then nothing will show in the log. There are other filters as well.

**Comment by Ingegneus** [comment]
yes but I printed the same part back to back with the plugin enabled and without. those filters aren't the problem I think. most messages during printing are the commands, and those appear and diappear randomly when the plugin is enabled

**Comment by jeffeb3** [comment]
Ok. I believe you. Thanks for the extra details.

**Comment by Ingegneus** [comment]
thanks for taking the time answering! really appreciate it
